src/Grid/Row/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, CSSProperties, ReactNode } from "react";
+import Styles from "./index.style";
+import classNames from "../../utils/classNames";
+
+type RowProps = {
+  children: ReactNode;
```

Comment:
Children doesnt need to be defined by us as it comes from the react Functional Component prop definition

Reply:
oops missed it 👀